python/servo/devenv_commands.py

```diff
@@ -67,11 +74,57 @@ def cargo_update(self, params=None, package=None, all_packages=None):
         help='Updates all packages. NOTE! This is very likely to break your ' +
              'working copy, making it impossible to build servo. Only do ' +
              'this if you really know what you are doing.')
-    def update_cargo(self, params=None, package=None, all_packages=None):
+    @CommandArgument(
+        '--dry-run', '-d', action='store_true',
+        help='Show outdated packages.')
+    def update_cargo(self, params=None, package=None, all_packages=None, dry_run=None):
         if not params:
             params = []
 
-        if package:
+        if dry_run:
```

Comment:
This is a lot of code that doesn't involve cargo at all, AFAICT, and handles things like major updates, which cargo wouldn't do. What's the reason for integrating this into the `update-cargo` command, rather than adding a new command for it?